src/util/secrets/secrets.c

```diff
@@ -1006,10 +1056,15 @@ errno_t sss_sec_get(TALLOC_CTX *mem_ctx,
     attr_enctype = ldb_msg_find_attr_as_string(res->msgs[0], "enctype", NULL);
```

Comment:
Attr name "enctype" is hard-coded everywhere. It seems other places in SSSD code use `define`

Reply:
The same applies to other attribute names.

Reply:
Ok. Added a new commit to address this.